base/uk.ac.stfc.isis.ibex.banner/src/uk/ac/stfc/isis/ibex/banner/BannerModel.java

```diff
@@ -0,0 +1,69 @@
+package uk.ac.stfc.isis.ibex.banner;
+
+import java.util.Collection;
+
+//import com.google.common.base.Strings;
+
+import uk.ac.stfc.isis.ibex.configserver.Configurations;
+import uk.ac.stfc.isis.ibex.configserver.configuration.BannerItem;
+import uk.ac.stfc.isis.ibex.epics.observing.BaseObserver;
+import uk.ac.stfc.isis.ibex.model.ModelObject;
+
+/**
+ * Contains the overall model of the banner in the form of an up-to-date
+ * collection of all banner items.
+ */
+public class BannerModel extends ModelObject {
+	
+	Collection<BannerItem> bannerItems;
+
+    /**
+     * Instantiates the object and starts observing the PV holding the banner's
+     * information.
+     */
+	public BannerModel() {
+		Configurations.getInstance().variables().bannerDescription.addObserver(descriptionAdapter);
+    }
+
+    private final BaseObserver<Collection<BannerItem>> descriptionAdapter = new BaseObserver<Collection<BannerItem>>() {
+
+        @Override
+        public void onValue(Collection<BannerItem> value) {
+            setBannerItems(value);
+            for (BannerItem item : value) {
+            	item.createPVObservable();
+            }
+        }
+
+        @Override
+        public void onError(Exception e) {
+            System.out.println("Error");
```

Comment:
What happens to the bump strip on error. Probably an error needs to appear in the banner.